Automated testing in WordPress/Requirements.md

```diff
@@ -0,0 +1,127 @@
+open sourced by => Epinions.com
+```
+
+Go to `/usr/local/etc/php/7.3/php.ini` and edit it by adding
+
+```bash
+zend_extension=/usr/local/Cellar/php/7.3.7/pecl/20180731/xdebug.so
+xdebug.remote_autostart=1
+xdebug.remote_port=9000
+xdebug.remote_enable=1
+```
+
+At the end of the `php.ini` file.
+
+That's it, you're ready to go.
+
+### Unsuccessful installation
+
+When installing xdebug, you might get the following error
+
+```bash
+Build process completed successfully
+Installing '/usr/local/Cellar/php/7.3.7/pecl/20180731/xdebug.so'
+
+Warning: mkdir(): File exists in System.php on line 294
+PHP Warning:  mkdir(): File exists in /usr/local/Cellar/php/7.3.7/share/php/pear/System.php on line 294
+
+Warning: mkdir(): File exists in /usr/local/Cellar/php/7.3.7/share/php/pear/System.php on line 294
+ERROR: failed to mkdir /usr/local/Cellar/php/7.3.7/pecl/20180731
+```
+
+Go to your php installation folder (`/usr/local/Cellar/php/7.3.7/` for instance) and type
+
+```bash
+ls -all
+
+total 168
+-rw-r--r--   1 infinum-denis  staff   3.1K Aug  7 10:10 INSTALL_RECEIPT.json
+-rw-r--r--   1 infinum-denis  staff   3.1K Jul  3 13:30 LICENSE
+-rw-r--r--   1 infinum-denis  staff    64K Jul  3 13:30 NEWS
+-rw-r--r--   1 infinum-denis  staff   1.6K Jul  3 13:30 README.md
+drwxr-xr-x  12 infinum-denis  staff   384B Aug  7 10:10 bin
+-rw-r--r--   1 infinum-denis  staff   628B Aug  7 10:10 homebrew.mxcl.php.plist
+drwxr-xr-x   3 infinum-denis  staff    96B Jul  3 13:30 include
+drwxr-xr-x   4 infinum-denis  staff   128B Jul  3 13:30 lib
+lrwxr-xr-x   1 infinum-denis  staff    23B Aug  7 10:10 pecl -> /usr/local/lib/php/pecl
+drwxr-xr-x   3 infinum-denis  staff    96B Jul  3 13:30 sbin
+drwxr-xr-x   4 infinum-denis  staff   128B Jul  3 13:30 share
+```
+
+You can see that `pecl` is symlinked to `/usr/local/lib/php/pecl`, but `which pecl` specified `/usr/local/bin/pecl`. Which is itself a symlink to `/usr/local/Cellar/php/7.3.7/bin/pecl`. So you need to remove the symlink
+
+```bash
+unlink pecl
+```
+
+Now you can install Xdebug again, and it should work like in the previous case (follow the steps described there to add it correctly in your `php.ini`).
+
+## Possible sideaffects
+
+Possible sideaffects of running Xdebug, especially if you enable profiler output in the xdebug settings in `php.ini` is that that file can grow very large, so just make sure that you delete it every once in a while, or just don't enable logging.
```

Comment:
A possible side effect of running Xdebug, especially if you enable profiler output in Xdebug settings in `php.ini`, is that that file can grow very large. To prevent this, just make sure you delete it every once in a while, or simply don't enable logging.